cmd/provider/main.go

```diff
@@ -83,7 +87,7 @@ func main() {
 	kingpin.FatalIfError(apis.AddToScheme(mgr.GetScheme()), "Cannot add Helm APIs to scheme")
 	o := controller.Options{
 		Logger:                  log,
-		MaxConcurrentReconciles: *maxReconcileRate,
+		MaxConcurrentReconciles: maxConcurrency,
```

Comment:
Just curious - why hardcode this instead of using the value from the --max-reconcile-rate input parameter?

Reply:
Max reconciliation rate caps the number of reconciliation that's allowed to happen at a given time, like 10 per second, whereas max concurrent reconciles controls how many reconciliations can be actively running in parallel at the same time, i.e. `don't process new events if there are 10 other go routines processing reconciles`. So, we can have another argument to control that but using the same argument for both is sort of misleading and the original number was `10`, so I kept it same.